### PR TITLE
Show news cards on the logged out landing page

### LINKED ISSUE #5127: Show recent news cards on landing page
Replace link to development blog with four small news cards.

### PR DESCRIPTION
**Pull request by LiquidPL**
Resolves #5127.

### CODE REVIEW

File: resources/assets/lib/news-index/post-item.tsx [hunk 1]
```diff
@@ -21,39 +21,46 @@ import { route } from 'laroute';
...
-  let cover;
+interface Props {
+  modifiers?: string[];
+  post: PostJson;
+}
 
-  if (post.first_image != null) {
-    cover = <img className='news-card__cover' src={post.first_image} />;
-  }
+export default class PostItem extends React.Component<Props, {}> {
```
Comment (notbakaneko) [inline resources/assets/lib/news-index/post-item.tsx]: not sure why this is now a class, it doesn't have a constructor, nor use any lifecycle methods other than `render`

File: resources/assets/less/bem/news-card.less [hunk 2]
```diff
@@ -36,6 +36,20 @@
     min-height: 150px;
   }
 
+  &--landing {
```
[the remark this answers is not in the document]
Comment (LiquidPL) [reply resources/assets/less/bem/news-card.less]: image

should I also make the cards less tall while we're at it, or are they fine like this?

[the remark this answers is not in the document]
Comment (nanaya) [reply resources/assets/less/bem/news-card.less]: the height is fine

File: resources/assets/less/bem/landing-news.less [hunk 3]
```diff
@@ -16,26 +16,25 @@
...
 
-    &:hover {
-      .thicker-box-shadow();
+    @media @desktop {
+      flex-direction: row;
+      margin: -10px;
     }
 
-    &--blog {
-      .at2x-simple('/images/landing/button-blog.png');
```
Comment (nanaya) [inline resources/assets/less/bem/landing-news.less]: sorry I missed this one. The referenced file should be removed as well.

**Pull request merged**